tools/UpdatePaths/Scripts/9994_knives.txt

```diff
@@ -0,0 +1,2 @@
+/obj/item/knife/ : /obj/item/kitchen/knife/{@OLD}
```

Comment:
Should this have the / at the end?

Reply:
Yes. Those are every knife besides kitchen knives.

/obj/item/kitchen/knife is a kitchen knife. Before it was the parent of all knives, but now it is a derivative of /obj/item/knife.

In the unlikely event a merge conflict occurs, I will fix it within the hour.